src/index.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash';
+import './style.css';
+import {homePage} from "./landing";
+import {nav} from "./navControl";
+import {menu} from "./menu"
+import {contact} from "./contact"
+
+
+
+
+const displayController = (()=>{
+   
+    let content = document.querySelector("#content");
+    
+    const render = ()=>{
+    
+        nav.render(content);
+        let menuButton = document.querySelector("#menu-but");
```

Comment:
- [ ] [OPTIONAL] Instead of selecting each menu button one by one and adding an event listener one by one again why don't you select them by their class, if it's the same, add the event to all inside a loop and then according to the one that was clicked, using `e.target.id` to check, for example, perform different DOM actions?
You could achieve the same result you have with a lot less code if you feel like refactoring.

Reply:
Sure, i'll definately look at this later on. Thank you.